workspaces/orchestrator/plugins/orchestrator-form-widgets/http-test-server/README.md

```diff
@@ -0,0 +1,50 @@
+# Development-only HTTP server
+
+The purpose of this Express Node.js server is to respond with JSON Schema chunks and other test data to ease development of the active widgets, like the `SchemaUpdater`.
+
+The production environment is expected to provide another HTTP server (unrelated to this testing one) dynamically providing `SchemaUpdater` snippets or fetch/validation functionality for other widgets.
+
```

Comment:
I think the documentation should include how to create the workflow that users the server, by running yarn "update-running-workflow" and mention that it'll only work with the dev mode

Reply:
Added a commit to show the direction. We will handle the documentation within [FLPATH-2291](https://issues.redhat.com/browse/FLPATH-2291)